glyphs-reader/src/font.rs

```diff
@@ -70,11 +70,21 @@ pub struct Font {
 }
 
 #[derive(Debug, PartialEq, Eq, Hash)]
-pub struct FeatureSnippet(String);
+pub struct FeatureSnippet {
+    pub content: String,
+    pub disabled: bool,
+}
 
 impl FeatureSnippet {
-    pub fn as_str(&self) -> &str {
-        self.0.as_str()
+    pub fn new(content: String, disabled: bool) -> Self {
+        FeatureSnippet { content, disabled }
+    }
+
+    pub fn str_if_enabled(&self) -> Option<&str> {
+        match self.disabled {
```

Comment:
Maybe `self.disabled.then_some(&self.content)`?

Reply:
Or, more correctly: `(!self.disabled).then_some…`

Reply:
You get a str when it's false so it would be (!self.disabled).then_some I suppose. It's less noisy, will update.